HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -164,10 +164,16 @@ def customiseFor2017DtUnpacking(process):
 
     return process
 
+def customizeForTauDiscriminatorDataFormatChanges(process):
+    for x in filters_by_type(process, "PFTauSelector"):
+        x.discriminatorContainers = cms.VPSet()
```

Comment:
Is it actually possible to have fillDescriptions for PFTauSelector beeing a typedef of ObjectSelectorStream<PFTauSelectorDefinition>, where PFTauSelectorDefinition is a struct?

Reply:
why not? 
```ObjectSelectorStream<PFTauSelectorDefinition>::fillDescriptions( ... ```

Reply:
That (but using the typedef, see last line below) is basically what I did and I got:
```
error: no member function 'fillDescriptions' declared in 'ObjectSelector<PFTauSelectorDefinition, std::vector<reco::PFTau>, NonNullNumberSelector, helper::NullPostProcessor<std::vector<reco::PFTau>, edm::stream::EDFilter<> >, helper::CollectionStoreManager<std::vector<reco::PFTau>, helper::IteratorToObjectConverter<std::vector<reco::PFTau> > >, helper::ObjectSelectorBase<std::vector<reco::PFTau>, edm::stream::EDFilter<> >, reco::modules::NoEventSetupInit<PFTauSelectorDefinition> >'
   static void PFTauSelector::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
```
It would require a declaration of fillDescriptions in the ObjectSelectorStream template, right?

Reply:
ah, right, I was wrong in the guessed simplicity. The following should work.

```c++
class PFTauSelector : public ObjectSelectorStream<PFTauSelectorDefinition> {
public:
  PFTauSelector(const edm::ParameterSet& ps ) : ObjectSelectorStream<PFTauSelectorDefinition>(ps) {}
  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
```

Reply:
Thank you, I'll try this